public/js/mainEditor.js

```diff
@@ -23,6 +23,16 @@ const navbarPosition = select(".navbarPosition");
 const navbarPreview = select(".cp-navbar-container");
 const paddingInputForNavbar = select('.paddingInputForNavbar')
 const generateCode = select('.generateCodeButton');
+const backToEditing = select('.backToEditing');
+const navEditContainer = select('.navEditContainer');
+const navPreview = select('.navPreview');
+const resultContainer = select('.resultContainer');
+const finalCssCode = select('.finalCssCode');
```

Comment:
Unused variable

Reply:
We will use them when we want to display the css and html code, should I remove them now???
@NoureldeanSaed